client/src/components/screens/PatientSubmission/components/ClinicalInformation/components/FamilyStorySection.tsx

```diff
@@ -145,18 +141,18 @@ const FamilyStorySection: React.FC<Props> = ({ familyHistoryResources }) => {
         <Col>
           <Form.Item
             label={intl.get('form.patientSubmission.clinicalInformation.familyHistory.familyHealth')}
-            initialValue={hasFamilyHealthCondition ? 'yes' : 'no'}
+            rules={[{ required: true, message: <ErrorText text="form.patientSubmission.clinicalInformation.validation.requiredField" /> }]}
```

Comment:
In my opinion `ErrorText` should receive the text to show. The translation should be done before and the text (FR or EN) should be passed to the component. Its a bit confusing, because it requires a text prop but receive the path for the translation.